frontend/index.html

```diff
@@ -204,7 +204,7 @@
 					</div>
 					<div class="field">
 						<label>Template:</label>
-						<div class="ui search selection dropdown" id="templatesDropdown">
+						<div class="ui multiple search selection dropdown" id="templatesDropdown">
 							<input name="templates" type="hidden">
```

Comment:
notice : 
Use multiple dropdown UI here, but actually, it can only allow single template selection by using "maxSelections:1" option. See app.js, line 235, 257